build/full-bundle.ts

```diff
@@ -33,65 +34,56 @@ import { getExternals } from './utils'
     external: getExternals({ full: true }),
   }
 
-  console.log(chalk.cyan('Start generating full bundle'))
-
-  const umd = {
-    format: 'umd',
-    file: path.resolve(buildOutput, 'element-plus/dist/index.js'),
-    exports: 'named',
-    name: 'ElementPlus',
-    globals: {
-      vue: 'Vue',
-    },
-  }
-
-  const umdMinified = {
-    ...umd,
-    file: path.resolve(buildOutput, 'element-plus/dist/index.full.js'),
-  }
+  cyan('Start generating full bundle')
 
-  console.log(chalk.bold(chalk.yellow('Building bundle')))
+  yellow(bold('Building bundle'))
 
   // Full bundle generation
   const bundle = await rollup.rollup({
     ...config,
     plugins: [...config.plugins, RollupResolveEntryPlugin()],
   })
 
-  console.log(chalk.yellow('Generating index.full.js'))
+  yellow('Generating index.full.js')
 
-  await bundle.write(umdMinified as any)
+  await bundle.write({
+    format: 'umd',
+    file: path.resolve(buildOutput, 'element-plus/dist/index.full.js'),
+    exports: 'named',
+    name: 'ElementPlus',
+    globals: {
+      vue: 'Vue',
+    },
+  })
 
-  console.log(chalk.green('index.full.js generated'))
+  green('index.full.js generated')
 
-  console.log(chalk.yellow('Generating entry files without dependencies'))
+  yellow('Generating entry files without dependencies')
 
   // Entry bundle generation
 
   const entryFiles = await fs.promises.readdir(epRoot, { withFileTypes: true })
 
   const entryPoints = entryFiles
     .filter((f) => f.isFile())
-    .filter((f) => {
-      return f.name !== 'package.json' && f.name !== 'README.md'
-    })
+    .filter((f) => !['package.json', 'README.md'].includes(f.name))
     .map((f) => path.resolve(epRoot, f.name))
 
   const entryBundle = await rollup.rollup({
     ...config,
     input: entryPoints,
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    external: (_) => true,
+    external: () => true,
   })
 
-  const rewriter = (id) => {
+  const rewriter = (id: string) => {
     if (id.startsWith(`${EP_PREFIX}/components`))
       return id.replace(`${EP_PREFIX}/components`, './components')
-    if (id.startsWith(EP_PREFIX) && excludes.every((e) => !id.endsWith(e)))
+    else if (id.startsWith(EP_PREFIX) && excludes.every((e) => !id.endsWith(e)))
       return id.replace(EP_PREFIX, '.')
+    else return id
```

Comment:
With `return` here there is actually no need to use `else`. Even though it is semantically correct.

Reply:
If don’t return, typescript will add `undefined` to the return value.